atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServices.java

```diff
@@ -107,15 +107,29 @@ static void waitForSchemaVersions(
                 sb.append(String.format("%n\tNode: %s", node));
             }
         }
+        sb.append(String.format("\nThe current keyspaces are: %s", client.describe_keyspaces()));
```

Comment:
This is not going to be a fast call. Do we really need it?

Reply:
This is going to happen if we don't reach a consensus on startup after changing the Cassandra schema. In this case, which shouldn't be often, I'd say it's better to have more logging and try to detect why the nodes diverged. If this becomes noisy, we can delete this line later.